### PR TITLE
Release Smart split items using mouse cursor context (also edit cursor, razor area and time selection) v3.20

### PR DESCRIPTION
**Pull request by AZpercussion**
- Added TCP context action
- fixed bug with empty takes
- sticked to imgui version 8.7.6 for stable work

### CODE REVIEW

File: Items Editing/az_Smart split items by mouse cursor.lua [hunk 1]
```diff
@@ -1,7 +1,10 @@
 -- @description Smart split items using mouse cursor context (also edit cursor, razor area and time selection)
 -- @author AZ
--- @version 3.15
--- @changelog - fixed one silly bug for new users
+-- @version 3.2
```
Comment (cfillion) [inline Items Editing/az_Smart split items by mouse cursor.lua]: Version numbers are not decimal numbers. 3.2 is a lot smaller than 3.15 (13 minor versions smaller) and so existing users would not receive the update.

Comment (AZpercussion) [reply Items Editing/az_Smart split items by mouse cursor.lua]: Ah, so will 3.20 work fine?

Comment (cfillion) [reply Items Editing/az_Smart split items by mouse cursor.lua]: Yes. Should I change it to 3.20 and merge?

Comment (AZpercussion) [reply Items Editing/az_Smart split items by mouse cursor.lua]: Yes, please)

**Pull request merged**